frontend/src/audios.html

```diff
@@ -8,8 +8,18 @@
   - topnav
 jsFile: audios.js
 ---
-<div class="mt-4 flex">
-  <div id="filter-panel" class="fixed top-0 z-20 hidden h-full w-64 max-w-full flex-col bg-white text-sm open:flex md:static md:z-auto md:flex md:w-1/5 md:max-w-sm">
+<div
+  class="mt-4 flex"
+>
+  <div
+    x-data="{open: false}"
+    @open-filter.window="open = $event.detail"
```

Comment:
It's not only for opening the filter. It's for closing also. That is why better to name the event something like `toggle-filter`.

Reply:
And for completeness, use `!open` as a fallback in case there is no `event.detail`.

Reply:
updated for fallback, i case `$event-detail` is undefined